.github/ISSUE_TEMPLATE/tensorflow_issue_template.yaml

```diff
@@ -72,80 +70,57 @@ body:
     id: OS
     attributes:
       label: OS Platform and Distribution
-      description: 
       placeholder: e.g., Linux Ubuntu 16.04
-    validations:
-      required: false
   - type: input
     id: Mobile
     attributes:
-      label: Mobile device
-      description: 
+      label: Mobile Device
       placeholder: e.g., Linux Ubuntu 16.04
-    validations:
-      required: false
   - type: input
     id: Python
     attributes:
-      label: Python version
-      description: 
+      label: Python Version
       placeholder: e.g., 3.9
-    validations:
-      required: false
   - type: input
     id: Bazel
     attributes:
-      label: Bazel version
-      description: if compiling from source
-      placeholder: 
-    validations:
-      required: false
+      label: Bazel Version
+      description: If compiling from source
   - type: input
     id: Compiler
     attributes:
-      label: GCC/Compiler version
-      description: if compiling from source
-      placeholder: 
-    validations:
-      required: false
+      label: GCC/Compiler Version
+      description: If compiling from source
   - type: input
     id: Cuda
     attributes:
-      label: CUDA/cuDNN version
-      description:
-      placeholder: 
-    validations:
-      required: false
+      label: CUDA/cuDNN Version
   - type: input
     id: Gpu
     attributes:
-      label: GPU model and memory
-      description: if compiling from source
-      placeholder: 
-    validations:
-      required: false
+      label: GPU Model and Memory
+      description: If compiling from source
   - type: textarea
     id: what-happened
     attributes:
-      label: Current Behaviour?
+      label: Current Behavior?
       description: Also tell us, what did you expect to happen?
       placeholder: Tell us what you see!
-      value: "A bug happened!"
     validations:
       required: true
   - type: textarea
     id: code-to-reproduce
     attributes:
-      label: Standalone code to reproduce the issue
-      description: Provide a reproducible test case that is the bare minimum necessary to generate the problem. If possible, please share a link to Colab/Jupyter/any notebook.
+      label: Standalone Code to Reproduce the Issue
+      description: Provide a reproducible test case at the bare minimum necessary to generate the problem. Please share a link to Colab, Jupyter, or any notebook.
```

Comment:
This should stay "that is". We want a test case which is the bare minimum, not at least one test case